adafruit_progressbar/verticalprogressbar.py

```diff
@@ -0,0 +1,173 @@
+# SPDX-FileCopyrightText: Copyright (c) 2020-2021 Brent Rubell for Adafruit Industries, Hugo Dahl
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`verticalprogressbar`
+================================================================================
+
+Dynamic progress bar widget for CircuitPython displays
+
+
+* Author(s): Brent Rubell, Hugo Dahl
+
+Implementation Notes
+--------------------
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+
+# imports
+from enum import Enum
+from . import ProgressBarBase
+
+
+class VerticalFillDirection(Enum):
+    """This enum is used to specify the direction in which the progress
+    bar's display bar should fill as the value represented increases."""
+
+    BOTTOM_TO_TOP = 0, "Fills from the bottom up toward the top"
+    DEFAULT = BOTTOM_TO_TOP, "Default fill direction (BOTTOM_TO_TOP)"
```

Comment:
I'm not sure if I understand how this direction `DEFAULT` is working. If I make a vertical progress bar without specifying the direction: 
```python
progress_bar = VerticalProgressBar(
    (x, y), (width, height), min_value=-10, max_value=60, value=10
)
```
I get this bar (set to `value=40` as in this screenshot):
![image](https://user-images.githubusercontent.com/2406189/108636891-a71e0b80-744d-11eb-8518-401f855b09c2.png)

But this one which specifically sets `BOTTOM_TO_TOP: 
```python
progress_bar = VerticalProgressBar(
    (x, y), (width, height), min_value=-10, max_value=60, value=10,
    direction=VerticalFillDirection.BOTTOM_TO_TOP
)
```
Ends up this way: 
![image](https://user-images.githubusercontent.com/2406189/108636926-eea49780-744d-11eb-9764-9811d6614081.png)

Reply:
That's peculiar. I'll check it to see what gives. I might have misunderstood the way `Enum` works in Python, and instinctively went with the C# way I know. I'll have a look and fix it up. Good catch!

Reply:
Switching from `enum.Enum` to `enum.IntEnum` seems to provide the equivalency needed, whereas the original `enum.Enum` treats each value as different, despite the assignment.